server/src/concepts/offer/controllers/index.js

```diff
@@ -0,0 +1,16 @@
+import { updateOfferFunc } from "../useCases/updateOffer";
+
+export const updateOffer = async (req, res) => {
+  try {
```

Comment:
firstly you should use offerValidation from model/OfferValidation here
and then trigger updateOfferFunc when we will know that request is valid.